kythe/extractors/extractors.bzl

```diff
@@ -23,15 +23,17 @@ def extractor_action(name, extractor, args, mnemonics, output, needs_output = Fa
       output: The output file template (string).
       data: A list of data dependencies (optional).
       tags: A list of build tags (optional).
+      env: A dict of environment variables (optional).
     """
     xa_name = name + "_extra_action"
 
+    env_vars = ["{}={}".format(item[0], item[1]) for item in env.items()]
```

Comment:
optional: `"{}={}".format(*item)`  or `"=".join(item)` also work :-)

Reply:
(although the latter can get grabby if both elements aren't already strings)

Reply:
Interesting, didn't know about that mechanism of tuples